src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.DefaultOutputPaths.targets

```diff
@@ -32,14 +32,14 @@ Copyright (c) .NET Foundation. All rights reserved.
     
     <BaseOutputPath Condition="'$(BaseOutputPath)' == ''">bin\</BaseOutputPath>
     <BaseOutputPath Condition="!HasTrailingSlash('$(BaseOutputPath)')">$(BaseOutputPath)\</BaseOutputPath>
-    <OutputPath Condition="'$(OutputPath)' == '' and '$(PlatformName)' == 'AnyCPU'">$(BaseOutputPath)$(Configuration)\</OutputPath>
-    <OutputPath Condition="'$(OutputPath)' == '' and '$(PlatformName)' != 'AnyCPU'">$(BaseOutputPath)$(PlatformName)\$(Configuration)\</OutputPath>
+    <OutputPath Condition="'$(OutputPath)' == '' and '$(PlatformName)' == 'AnyCPU'">$(BaseOutputPath)$(Configuration).ToLowerInvariant()\</OutputPath>
```

Comment:
This should be `$(Configuration.ToLowerInvariant())` instead of `$(Configuration).ToLowerInvariant()` (and likewise in the other locations).

FYI, you should be able to just run `build` from a command prompt in the repo folder to build and run all the tests.